content/develop/connect/insight/release-notes/v.2.50.0.md

```diff
@@ -0,0 +1,23 @@
+---
+Title: Redis Insight v2.50.0, May 2024
+linkTitle: v2.50.0 (May 2024)
+date: 2024-05-30 00:00:00 +0000
+description: Redis Insight v2.50
+weight: 1
+
+---
+## 2.50 (May 2024)
+This is the General Availability (GA) release of Redis Insight 2.50.
+
+### Highlights
+- Optimize your Redis learning journey by signing in to your Redis Cloud account directly from Redis Insight. Easily provision a free Cloud database and level up your Redis expertise with our interactive tutorial. Or swiftly import databases from your Redis Cloud account to seamlessly proceed your journey in Redis Insight.
+- Explore common Redis use cases through a paired-up tutorial opened after uploading sample data for empty databases.
+- Performance and UX enhancements for JSON data structure in Browser
```

Comment:
```suggestion
- Performance and UX enhancements for JSON data structures in the Browser.
```

Reply:
let's use "the JSON data structure" since it is the only one